spec/features/task_list_spec.rb

```diff
@@ -28,7 +28,7 @@
     end
 
     it "shows a column for reviewed tasks" do
-      expect(page).to have_text "Reviewed"
+      expect(page).to have_text "&check;"
```

Comment:
I'm not sure why the reviewed test was still there—but that's not really a thing anymore, is it? We only have "approved" now, yes?

Reply:
Totally understandable.